.gitignore

```diff
@@ -38,5 +38,8 @@ pip-log.txt
 sdist/
 target/
 var/
+local/
 node_modules
 .DS_Store
+Makefile
+manage.py
```

Comment:
manage.py is actually a bash script with ``python -m shuup_workbench $@`` and the Makefile is a symlink to this file https://github.com/shuup/shuup-tools/blob/master/config/Makefile

I didn't know about the .git/info/exclude trick, that's very useful, thanks!